tcms/utils/tests/test_assign_permissions.py

```diff
@@ -0,0 +1,44 @@
+from django.contrib.auth.models import Group, Permission
+from django.test import TestCase
+
+from tcms.utils.permissions import assign_default_group_permissions
+
+
+class TestAssignDefaultGroupPermissions(TestCase):
+    """Test utils.permissions.assign_default_group_permissions()"""
+
+    def setUp(self):
+        self.admin = Group.objects.get(name='Administrator')
+        self.admin.permissions.all().delete()
+        self.tester = Group.objects.get(name='Tester')
+        self.tester.permissions.all().delete()
+
+    def test_administrator_has_all_permissions(self):
+        assign_default_group_permissions()
+        self.assertEqual(Permission.objects.all().count(),
+                         self.admin.permissions.count())
+
+    def test_tester_has_kiwi_apps_permissions(self):
+        assign_default_group_permissions()
+        for app_name in ['bugs', 'django_comments', 'linkreference', 'management',
+                         'testcases', 'testplans', 'testruns']:
+            self.assertEqual(set(Permission.objects.filter(
+                content_type__app_label__contains=app_name).values_list(
+                'codename', flat=True)),
+                set(self.tester.permissions.filter(
+                    content_type__app_label__contains=app_name).values_list(
+                    'codename', flat=True)),
+                msg='Tester misses some %s permissions' % app_name)
+
+        self.assertEqual(set(Permission.objects.filter(
+                content_type__app_label__contains='attachments').exclude(
+                codename='delete_foreign_attachments').values_list(
+                'codename', flat=True)),
+            set(self.tester.permissions.filter(
+                content_type__app_label__contains='attachments').values_list(
+                'codename', flat=True)),
+            msg='Tester misses some attachments permissions')
+
+        self.assertEqual(0, self.tester.permissions.filter(
+                content_type__app_label='attachments',
+                codename='delete_foreign_attachments').count())
```

Comment:
.exists() with True/False assertion is more pythonic, otherwise this isn't wrong.